docs/core/testing/unit-testing-mstest-runner-integrations.md

```diff
@@ -0,0 +1,51 @@
+---
+title: MSTest runner integrations
+description: Learn how to run MSTest runner tests through MSBuild and dotnet test.
+author: nohwnd
+ms.author: jajares
+ms.date: 12/20/2023
+---
+
+# MSTest runner integrations
+
+MSTest runner integrates with MSBuild and `dotnet test` to provide a simple way to run all tests in a solution (`.sln`) or in a single project.
+
+## `dotnet test` integration
+
+### `dotnet test` integration - VSTest mode
+
+MSTest runner provides a compatibility layer to work with `dotnet test` seamlessly. This layer requires `Microsoft.Testing.Platform.MSBuild` NuGet package. This package is automatically installed as a dependency of `MSTest` and `MSTest.TestAdapter` package.
+
+Tests can be executed by running:
+
+```
+dotnet test
+```
+
+This layer runs test through VSTest and integrates with it on VSTest Test Framework Adapter level.
+
+### `dotnet test` - MSTest runner mode
+
+By default VSTest is used to execute MSTest runner tests. User can opt-in to a fully MSTest runner provided mode. This mode integrates directly with the `dotnet test` target at MSBuild target level. The user opts-in by specifying `<TestingPlatformDotnetTestSupport>true</TestingPlatformDotnetTestSupport>` in their csproj file. When this mode is enabled, VSTest is not used to run tests at all.
```

Comment:
This is true also for VB.NET and F# so I would avoid saying `csproj`.

Reply:
```suggestion
By default VSTest is used to execute MSTest runner tests. User can opt-in to a fully MSTest runner provided mode. This mode integrates directly with the `dotnet test` target at MSBuild target level. The user opts-in by specifying `<TestingPlatformDotnetTestSupport>true</TestingPlatformDotnetTestSupport>` in their project file. When this mode is enabled, VSTest is not used to run tests at all.
```